Helper/Data.php

```diff
@@ -149,4 +161,8 @@ public function getMinInstallmentValue($country)
 
         return $configValue;
     }
+
+    public function getModuleVersion() {
+        return $this->_moduleList->getOne(self::MODULE_NAME)['setup_version'];
```

Comment:
Would this work? This way we don't need a constant for this
```suggestion
        return $this->_moduleList->getOne($this->_getModuleName())['setup_version'];
```